eyeball-im-util/src/vector/ops.rs

```diff
@@ -1,71 +1,75 @@
-use std::collections::VecDeque;
-
+use arrayvec::ArrayVec;
 use eyeball_im::VectorDiff;
 
-pub trait VectorDiffContainerOps<T> {
+pub trait VectorDiffContainerOps<T>: Sized {
     type Family: VectorDiffContainerFamily;
+    type LimitBuf: Default;
```

Comment:
We may want to rename `LimitBuf` to `Buffer`, so that it's less tied to `Limit`. It can have another usage for another adapter.